src/Tests/Functions.fs

```diff
@@ -787,4 +787,34 @@ let tests =
                 let vnetConnections = resources |> getResource<Web.VirtualNetworkConnection>
                 Expect.hasLength vnetConnections 1 "incorrect number of Vnet connections"
             }
+            test "Supports .NET framework 4.8 in netFramework field" {
+                let app =
+                    functions {
+                        name "net6"
+                        use_runtime FunctionsRuntime.DotNetFramework48
+                    }
+
+                let site = app |> getResources |> getResource<Web.Site> |> List.head
+                Expect.equal site.NetFrameworkVersion.Value "v4.0" "Wrong dotnet version"
+            }
+            test "Supports .NET 6 in netFramework field" {
+                let app =
+                    functions {
+                        name "net6"
+                        use_runtime FunctionsRuntime.DotNet60
+                    }
+
+                let site = app |> getResources |> getResource<Web.Site> |> List.head
+                Expect.equal site.NetFrameworkVersion.Value "v6.0" "Wrong dotnet version"
+            }
+            test "Supports .NET 7 netFramework field" {
```

Comment:
You can probably merge these two tests into a single list comprehension as they only differ by a single parameter.